src/book.hh

```diff
@@ -24,8 +24,8 @@ namespace tagt::xchg::book {
 struct Book {
   size_t next_order_id { 0 };
   std::vector<order::Id> order_ids;
-  std::unordered_map<order::Id, std::shared_ptr<order::Order>> orders;
-  std::unordered_map<order::Ticker, order::Price> last_price;
+  std::unordered_map<order::Id, std::shared_ptr<order::Order>> orders;/**maps Id to shared_ptr*/
+  std::unordered_map<order::Ticker, order::Price> last_price;/**maps Ticker to Price */ 

```

Comment:
Also, don't mean to get repetitive but I feel like this is one of those comments that doesn't really tell you anything new other than looking at the code. I'd like to explain the why here rather than the what. I.e. why am I tracking the last price? How does the code take care of that? Why are we using these data structures? etc.